packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -136,11 +154,18 @@ export const createRdsLambda = (
     ],
     lambdaRole,
     environment,
-    undefined,
+    Duration.seconds(30),
     stack,
+    sqlLambdaVpcConfig,
```

Comment:
This is the intended change on this file. Most of the other changes are lint fixes.